app.js

```diff
@@ -1,183 +1,59 @@
+Stores.prototype.storeRandomCust = function () {
+  return Math.random() * (this.maxCust - this.minCust) + this.minCust;
+};
+Stores.prototype.storeHourlySales = function() {
+  for (var i = 0; i < hours.length; i++){
+    var randCookies = Math.floor(this.avgCookie * this.storeRandomCust());
+    this.hourSales.push(randCookies);
+    this.totalSales += randCookies;
     }
-  }
-  render: function () {
-    this.storeHourlySale();
-    var sectionEl = document.getElementById('capHill');
-    sectionEl.textContent = 'Capital Hill Store';
-    var ulEl = document.createElement('ul');
+};
+Stores.prototype.render = function() {
+  this.storeHourlySales();
+  var storeEl = document.getElementById('shops');
+  var newStoreEl = document.createElement('section');
+  storeEl.appendChild(newStoreEl);
+  var ulEl = document.createElement('ul');
     for (var i = 0; i < hours.length; i++) {
       var liEl = document.createElement('li');
       liEl.textContent = hours[i] + this.hourSales[i];
-      ulEl.appendedChild(liEl);
-    }
-    var liElTotal = document.createElement('li');
-    liElTotal.textContent = 'Total Sales: ' + this.totalSales  ulEl.appendedChild(liElTotal);
-    sectionEl.appendedChild(ulEl);
-  }
-}
-
-var alki {    // Alki store
-  minCust: 2;
-  maxCust: 16;
-  avgCookie:4.6;
-  totalSales: 0;
-  hourSales: [];
-
-  storeRandCust: function () {  // Random Number
-    return Math.random() * (this.maxCust - this.minCust) + this.minCust;
+      ulEl.appendChild(liEl);
+  };
+
+  var liElTotes = document.createElement('li');
+  liElTotes.textContent = 'Total Sales: ' + this.totalSales
+  ulEl.appendChild(liElTotes);
+  newStoreEl.textContent = this.name;
+  newStoreEl.appendChild(ulEl);
+};
 
-  }
 
-  storeHourlySale: function () { // Hourly Sales
-    for (var i = 0; i < hours.length; i++) {
-      var randCookie = Math.floor(this.avgCookie * this.storeRandCust());
-      this.hourSales.push(randCookie);
-      this.totalSales += randCookie;
-    }
+  var pikeStore = newStore('Pike Place','23','65','6.3');
+  var seaTac = newStore('SeaTac','3','24','1.2');
+  var seattleCenter = newStore('Seattle Center','11','38','3.7');
+  var capHill = newStore('Capital Hill','20','38','2.3');
+  var alki = newStore('Alki','2','16','4.6');
 
-    render: function () {
-      this.storeHourlySale();
-      var sectionEl = document.getElementById('alki');
-      sectionEl.textContent = 'Alki Store';
-      var ulEl = document.createElement('ul');
-      for (var i = 0; i < hours.length; i++) {
-        var liEl = document.createElement('li');
-        liEl.textContent = hours[i] + this.hourSales[i];
-        ulEl.appendedChild(liEl);
-      }
-      var liElTotal = document.createElement('li');
-      liElTotal.textContent = 'Total Sales: ' + this.totalSales
-      ulEl.appendedChild(liElTotal);
-      sectionEl.appendedChild(ulEl);
-    }
+  //might render stores
+  for(int i = 0; i< nameStores, i++){
+    this.render();
   }
```

Comment:
I'm getting quite a few linter errors as well. Resubmit when you get further along and we can take another look but so far it's looking pretty good.